### PR TITLE
Fix circular references (fixed and tested on all examples)

### PR DESCRIPTION
**Pull request by Itee**
Next to #10627 

Fixed missing accessors management !

Tested on all example without any error !

P.S: In one PR, due to past, for the last time.

### CODE REVIEW

**Comment by TristanVALCKE** [comment]
I'm resolving conflicts... with previous modified files

**Comment by TristanVALCKE** [comment]
Conflicts resolved ! Ready to be merge.

And all my apology about previous accessors bug !!!

File: src/core/BufferAttribute.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 }
 
+Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {
+
+	set: function(value) { if ( value === true ) this.version ++; }
+
+});
```
Comment (WestLangley) [inline src/core/BufferAttribute.js]: Friendly tip: before you do too many of these, be aware of your formatting. :)
```javascript
Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {

	set: function ( value ) {

		if ( value === true ) this.version ++;

	}

} );
```
http://zz85.github.io/mrdoobapproves/

Comment (TristanVALCKE) [inline src/core/BufferAttribute.js]: It is a setter for one lined function, but yes i can reformat this correctly. And in fact it seems that all my change were erase by merging... except the conflict files -_-' ! Damned

I close this one the time to got a stable version (with good formating ;)

Comment (WestLangley) [inline src/core/BufferAttribute.js]: No big deal. We can fix the formatting... Getting it right the first time just makes less work for others... Thank you for your efforts.  :)

**Pull request closed without merging**